eos-autoupdater/docs/eos-autoupdater.conf.5

```diff
@@ -63,6 +63,14 @@ random interval, uniformly distributed between 0 and the number of days set in
 this option. It must be \fI0\fP or greater. An interval of \fI0\fP days means
 that the wait is always exactly \fIIntervalDays=\fP long.
 .\"
+.IP "\fIUserVisibleUpdateDelayDays=\fP"
+.IX Item "UserVisibleUpdateDelayDays="
+Number of days to wait until applying a user visible update. If
+\fBeos\-autoupdater\fP(8) is triggered earlier than this number of days
+since a user visible update was seen, it will exit. It must be \fI0\fP
+or greater. An interval of \fI0\fP days means that user visible updates
+will be applied immediately.
```

Comment:
Might also be worthwhile mentioning that `--force-update` will cause this setting to be ignored

Reply:
Good call.